src/__tests__/networks.integration.test.ts

```diff
@@ -9,14 +9,45 @@ describe('ethrResolver (alt-chains)', () => {
   const { address } = interpretIdentifier(addr)
   const checksumAddr = address
 
-  describe('eth-testnets', () => {
-    it('resolves on ropsten when configured', () => {
+  describe('eth-networks', () => {
+    it('resolves on mainnet with versionId', async () => {
+      const resolver = new Resolver(getResolver({ infuraProjectId: '6b734e0b04454df8a6ce234023c04f26' }))
+      const result = await resolver.resolve('did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388?versionId=12090174')
+      expect(result).toEqual({
+        didDocumentMetadata: {
+          nextVersionId: '12090175',
+          nextUpdate: '2021-03-22T18:14:29.000Z',
+        },
+        didResolutionMetadata: {
+          contentType: 'application/did+ld+json',
+        },
+        didDocument: {
+          '@context': [
+            'https://www.w3.org/ns/did/v1',
+            'https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/lds-ecdsa-secp256k1-recovery2020-0.0.jsonld',
+          ],
+          id: 'did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388',
+          verificationMethod: [
+            {
+              id: 'did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller',
+              type: 'EcdsaSecp256k1RecoveryMethod2020',
+              controller: 'did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388',
+              blockchainAccountId: '0x26bF14321004e770E7A8b080b7a526d8eed8b388@eip155:1',
+            },
+          ],
+          authentication: ['did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388#controller'],
```

Comment:
shouldn't this also include `assertionMethod` now?

Reply:
right. once I rebase this on to the latest changes, it will be.
good catch!

Reply:
done